dotcom-rendering/src/web/components/Liveness.importable.tsx

```diff
@@ -117,6 +119,7 @@ function getKey(
 			'filterKeyEvents',
 			filterKeyEvents ? 'true' : 'false',
 		);
+		if (activeTopic) url.searchParams.set('activeTopic', activeTopic);
```

Comment:
Is this `activeTopic` query param semantically the same as the other `topics` query param? If so, it might be better to be consistent?

Reply:
I think it is. 

There is a **topics** field in the CAPI article to represent all existing topics on that article.
And an **activeTopic** to represent the current selected topic. 

Elsewhere though we've been naming the parameter "topics", so I think it's in those places we might have to rename to "activeTopic". And probably stick to the singular until we allow selecting multiple. 

@marjisound @joecowton1 @abeddow91 what do you think?

Reply:
Maybe better still we should have the model contain "availableTopics" and "selectedTopics", then the parameter could always be "topics", to keep it concise.

Reply:
This way we don't have to rename the parameter again either :)

Reply:
I think that makes sense

> Maybe better still we should have the model contain "availableTopics" and "selectedTopics", then the parameter could always be "topics", to keep it concise.

I think that makes sense. Anna and Marjan are both of today but I'll update this and the corresponding Frontend PR